app/lib/efile/submission_error_parser.rb

```diff
@@ -45,11 +46,13 @@ def persist_errors_from_raw_response_metadata
       as_xml.search("ValidationErrorGrp").each do |error_group|
         error = EfileError.find_or_create_by!(
           code: error_group.at("RuleNum")&.text,
-          message: error_group.at("ErrorMessageTxt")&.text,
           category: error_group.at("ErrorCategoryCd")&.text,
           severity: error_group.at("SeverityCd")&.text,
           source: "irs"
         )
+        message = error_group.at("ErrorMessageTxt")&.text
```

Comment:
So just for my own learning: these are errors coming back from the irs and the other place you changed are errors coming back from other integrations (like the usps or whatever we do for addresses)? So theoretically any of them could have the code stay the same but the message text change?